src/dune/compilation_context.mli

```diff
@@ -10,6 +10,10 @@ open Import
     associated to each library, executable and executables stanza. *)
 type t
 
+type opaque =
```

Comment:
It may be worth adding a short comment explaining what the `opaque` setting means.

Reply:
(And from which "settings" it can be inherited.)

Reply:
Done